### ISSUE TITLE
Regression: legacy cannot create a new collection

### ISSUE THREAD

**Issue by kerwinso** [Rhaptos/Products.RhaptosModuleEditor#18]
found by Ryan S. on content03, reproduced on legacy-staging.cnx.org. 

Trying to create a new collection in legacy returns the following error regardless of what's in the `Summary` field:

```
Summary contains invalid CNXML.
fatal: exception "java.net.ConnectException" thrown: Connection refused (Connection refused)
```

![image](https://user-images.githubusercontent.com/11298165/39492902-b1f66b18-4d56-11e8-9a4c-77145e8e736c.png)

Trello card: https://trello.com/c/ZDospmQs/493-regression-legacy-cannot-create-a-new-collection

**Comment by kerwinso** [comment]
Re-opening to verify fix.

**Comment by kerwinso** [comment]
Verified fixed as of RhaptosModuleEditor 0.91. The glorious result is here: https://legacy-staging.cnx.org/content/m67363/latest/?collection=col24578/latest
